recipes/masurca/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+c_compiler:
```

Comment:
Is that really needed?

Reply:
It could be my conda build environment, but I found I got errors depending on the version of compiler and flags I used. It proved tricky to get it right.